DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/mvdr_beamforming.vcxproj.user

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file really necessary? Almost all code samples have deleted this file, as it was unnecessary and adds clutter.

Reply:
We need this file to specify the --in argument when invoking the executable.